config/api.yml

```diff
@@ -4674,6 +4674,10 @@
         :identifier: vm_scan
       - :name: simulate_policy
         :identifier: policy_simulation
+      - :name: associate
+        :identifier: vm_edit
+      - :name: disassociate
+        :identifier: vm_edit
```

Comment:
Do you need to add this to the collection actions too?  See line 4521 above.  

Notice how many are in both resource action and collection actions:  search for `:name: shutdown_guest` above